### ISSUE TITLE
Mobile Chrome "Add to Homescreen" icon background

### ISSUE THREAD

**Issue by jochec** [keeweb/keeweb#802]
Application icon shortcut have white background. Is it possible to make it transparent, so the icon is only circle?

Mobile Chrome version 62.0.3202.84
Stock Android 7.1.2 (with default launcher)

![screenshot_20171210-081423](https://user-images.githubusercontent.com/7040084/33803176-ab2cd2aa-dd89-11e7-89b7-44f43ec72821.png)

**Comment by antelle** [comment]
Yeah this is the expected behavior for iOS. Probably should be customized for platforms where icons can be clipped.
<img width="272" alt="apps" src="https://user-images.githubusercontent.com/633557/33803214-92ad35f2-dd8a-11e7-9c2c-a72811075793.png">

**Comment by meduzen** [comment]
I also encountered this issue as I just pinned KeeWeb on my Android phone home screen.

I’d like to take care of this issue in order to bring to Keeweb every icons and usual files/declaration currently missing.
<details>
<summary>This includes…</summary>
… some HTML declarations in `<head>` and a couple of files (various icon sizes and formats, `manifest.json` with better icon declaration and browserconfig.xml for Windows 8, 8.1 and 10). 
</details>

### What I need to do that

I need the highest quality icon you have for the KeeWeb logo (= the one on the screenshot with the blue gradient and the grey border). A PNG (ideally 512 px square) or any vectorial asset will be fine. Can I find it somewhere?

**Comment by antelle** [comment]
@meduzen you can use https://realfavicongenerator.net, however it generates too many icons, it needs some cleanup. Please try to avoid unnecessary icons, i.e. add only those required, without adding everything possible this generator adds, to keep external icons amount as small as possible. Also, some icons are inlined into html, please leave it there. New icons can be put into `icons` folder.
Please do not add manifest.json for now, it will be added once we migrate to pwa. afaik it's not required for favicon, so it's another issue.
Here's the icon in 512x512: ![icon512](https://user-images.githubusercontent.com/633557/35765295-efa376c0-08c0-11e8-9c6b-617c3a00ff5e.png)

**Comment by meduzen** [comment]
No worry, I use Real Favicon Generator for every project (I’m a front-end developer) and precisely knows what icon bits are useful or not.

For the `manifest.json`, I recommend to already add it in the HTML `<head>` now: you don’t have to fully embrace all the PWA big things (like Service Workers #156 or progressive enhancement for all the features) to get a working and useful `manifest.json` that brings a little more consistency to the UI.

And, bonus point: the Apache Manifest and the Web Manifest can live side by side as long as there’s no Service Worker running.

As Web Manifest is in the W3 spec (working draft, but adoption grows), it means it’s the right way to declare icons. Other options like declarations in `<head>` or the Windows `browserconfig.xml` file are just fallbacks. Using it helps consistency + removes bits from the HTML `<head>`.

If you absolutely don’t want to bring the `manifest.json` in the HTML, I recommend to keep this issue on hold and to do the job (even if it’s a small one) only once as the Web Manifest support landscape may change until then.

What do you think?

**Comment by antelle** [comment]
I would start with adding icons first, it makes more sense now. Probably we should add manifest.json too, you're right, it won't do any harm anyway.
